src/app/lib/utilities/preprocessor/rules/addIdsToItems.test.js

```diff
@@ -0,0 +1,106 @@
+import addIdsToItems from './addIdsToItems';
+
+jest.mock('uuid', () => () => 'mockId');
+
+const noId = {
+  metadata: {},
```

Comment:
Do we need `metadata`, `promo` and `relatedContent` to test this?

Reply:
No, they are not needed. Removed them